websites/A/AniChart/presence.ts

```diff
@@ -0,0 +1,36 @@
+const presence = new Presence({
+  clientId: "795125406264066099"
+}),
+presenceData: PresenceData = {
+  largeImageKey: "anichart"
+};
+
+presence.on("UpdateData", async () => {
+  presenceData.startTimestamp = Math.floor(Date.now() / 1000);
```

Comment:
i said to assign this startTimestamp to presenceData object. Or your startTimestamp msut change on every subpage?

Reply:
oh lmao ok